src/platform/datapath_raw_xdp_win.c

```diff
@@ -1284,14 +1284,18 @@ CxPlatDpRawUpdateConfig(
 
 _IRQL_requires_max_(PASSIVE_LEVEL)
 QUIC_STATUS
-CxPlatSocketUpdateQeo(
-    _In_ CXPLAT_SOCKET* Socket,
+RawSocketUpdateQeo(
+    _In_ CXPLAT_SOCKET_RAW* Socket,
     _In_reads_(OffloadCount)
         const CXPLAT_QEO_CONNECTION* Offloads,
     _In_ uint32_t OffloadCount
     )
 {
-    XDP_DATAPATH* Xdp = (XDP_DATAPATH*)Socket->Datapath;
+    if (!Socket->RawDatapath) {
```

Comment:
Is this an expected path that can/would be taken? Or can this be a debug assert instead?

Reply:
This is not needed anymore... I expected this was kind of debug assert, but covered by wrapper in datapath_win.c